rust/src/data_frame.rs

```diff
@@ -223,6 +223,65 @@ ocaml_export! {
         Abstract(data_frame.null_count()).to_ocaml(cr)
     }
 
+    fn rust_data_frame_fill_null_with_strategy(
+        cr,
+        data_frame: OCamlRef<DynBox<DataFrame>>,
+        strategy: OCamlRef<FillNullStrategy>
+    ) -> OCaml<Result<DynBox<DataFrame>,String>> {
+        let Abstract(data_frame) = data_frame.to_rust(cr);
+        let PolarsFillNullStrategy(strategy) = strategy.to_rust(cr);
+
+        data_frame.fill_null(strategy)
+        .map(Abstract).map_err(|err| err.to_string())
+        .to_ocaml(cr)
+    }
+
+    fn rust_data_frame_interpolate(
+        cr,
+        data_frame: OCamlRef<DynBox<DataFrame>>,
+        method: OCamlRef<InterpolationMethod>,
+    ) -> OCaml<Result<DynBox<DataFrame>,String>> {
+        let Abstract(data_frame) = data_frame.to_rust(cr);
+        let PolarsInterpolationMethod(method) = method.to_rust(cr);
+
+        // TODO: I expected there to be a way to get an iterator over a series
+        // directly, but I couldn't find anything; if this doesn't exist perhaps
+        // this should be added to upstream.
+        data_frame.get_column_names().into_iter().map(|column_name| {
+            data_frame.column(column_name).map(|series|
+                interpolate(&series, method)
+            )
+        })
```

Comment:
https://docs.rs/polars/latest/polars/frame/struct.DataFrame.html#method.get_columns ?

Reply:
Ugh, I can't believe I missed this, fixing now... 🤦